common/src/main/java/com/wynntils/commands/WynntilsCommand.java

```diff
@@ -79,9 +82,14 @@ private int reload(CommandContext<CommandSourceStack> context) {
             feature.disable();
         }
 
-        WebManager.reset();
+        // FIXME: This probably does not do what it was intended to do.
+        // reset
         ItemProfilesManager.reset();
-        WebManager.init(); // reloads api urls as well as web manager
+        // reloads api urls as well as web manager
```

Comment:
I intend to leave this more or less as-is, for this PR. I have opened https://github.com/Wynntils/Artemis/issues/824 to track how to really handle this.